pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1035,7 +1035,7 @@ func (n *NetworkProvider) reconcileStandaloneVPCSubnets(ctx context.Context, log
 	for _, sub := range subs {
 		logger.Infof("validating subnet %s", *sub.SubnetId)
 		if !tagsContains(ec2TagsToGeneric(sub.Tags), defaultAWSPrivateSubnetTagKey, "1") ||
-			!tagsContains(ec2TagsToGeneric(sub.Tags), aws.StringValue(croClusterOwnerTag.Key), aws.StringValue(croClusterOwnerTag.Value)) ||
+			!tagsContains(ec2TagsToGeneric(sub.Tags), *aws.String(croClusterOwnerTag.key), *aws.String(croClusterOwnerTag.key)) ||
```

Comment:
`croClusterOwnerTag.key` here is a `string`

`aws#String(string) *string` takes in a `string` and returns a pointer to it

The `*` then dereferences the pointer that `aws#String` returned, and provides back the original `string`

So we end up getting `croClusterOwnerTag.key` back. Unless I've misread, this should be the same as doing:

```suggestion
			!tagsContains(ec2TagsToGeneric(sub.Tags), croClusterOwnerTag.key, croClusterOwnerTag.value) ||
```

If that makes sense?

I've also changed the second `croClusterOwnerTag.key` to `croClusterOwnerTag.value` as I'm assuming that's what was intended?